MEEW-Client/MEEW-Client/Screens/Select/Cell/SelectCharacterCVC.swift

```diff
@@ -23,11 +23,18 @@ class SelectCharacterCVC: UICollectionViewCell {
     
     func setUI() {
         self.layer.cornerRadius = 16
+        characterImageView.layer.cornerRadius = 16
     }
```

Comment:
다른 클래스에서 작성한 코드 스타일과의 통일성을 위해서 다음 메서드들도 private하게 접근 제어 수준을 변경해주면 좋을 것 같습니다.
```suggestion
    private func setUI() {
        self.layer.cornerRadius = 16
        characterImageView.layer.cornerRadius = 16
    }
```